README.md

```diff
@@ -9,8 +9,173 @@ Tablesaw parquet I/O
 
 ## Overview
 
-__tablesaw-parquet__  is a [parquet](https://github.com/apache/parquet-mr) reader and writer for the [Tablesaw](https://github.com/jtablesaw/tablesaw) project.
+__tablesaw-parquet__  is an [Apache Parquet](https://parquet.apache.org/) reader and writer for the [Tablesaw](https://github.com/jtablesaw/tablesaw) project implemented using [parquet-mr](https://github.com/apache/parquet-mr).
 
-## Status
+## Versioning
 
-__tablesaw-parquet__  is currently under development.
\ No newline at end of file
+__tablesaw__  and  __tablesaw-parquet__  have different release schedules and versioning schemes. New  __tablesaw-parquet__  features will be available for the most recent *and* for the previous released version of  __tablesaw__ . Older tablesaw releases will not get updates.
+
+The first supported  __tablesaw__  version is  __v0.38.2__ .
+
+__tablesaw-parquet__  follows the [semantic versioning](https://semver.org/) scheme.
+
+## Getting started
+
+Add tablesaw-core and tablesaw-parquet to your project as follows:
+
+__maven:__
+
+```xml
+<properties>
+    <tablesaw.version>0.38.2</tablesaw.version>
```

Comment:
Specific version numbers for clarity sake

Reply:
Good point, thanks